substrate/frame/support/src/lib.rs

```diff
@@ -909,7 +909,7 @@ pub mod pallet_prelude {
 			Task, TypedGet,
 		},
 		Blake2_128, Blake2_128Concat, Blake2_256, CloneNoBound, DebugNoBound, EqNoBound, Identity,
-		PartialEqNoBound, RuntimeDebugNoBound, Twox128, Twox256, Twox64Concat,
+		PalletId, PartialEqNoBound, RuntimeDebugNoBound, Twox128, Twox256, Twox64Concat,
```

Comment:
You're not supposed to change this pallet in the context of migration to the umbrella crate

Reply:
Otherwise, I should depends on frame-support and `use frame_support::Pallet`. I think this is a part of migration to the umbrella crate.

Reply:
You should add `frame_support::PalletId` in a prelude in the umbrella crate (file mentioned in the other comment) and avoid changing this pallet.

Reply:
Make sense.

But after some thoughts, I think it's also good to place the `PalletId` here.

I added `PalletId` to frame-support's root 3 years ago https://github.com/paritytech/substrate/pull/8477. At that time, there wasn't a really good place for it. I think it's a good time to move it into `frame_support::pallet_prelude`, it is being used frequently.